pluginClientSdkLib/build.gradle

```diff
@@ -78,6 +80,8 @@ uploadArchives {
             }
             // Exclude support.v4, making it an optional dependency
             pom.whenConfigured { pom -> pom.dependencies.removeAll {dep -> dep.groupId == 'com.android.support' && dep.artifactId == 'support-v4' } }
+            // Exclude appcompat-v7, making it an optional dependency
+            pom.whenConfigured { pom -> pom.dependencies.removeAll {dep -> dep.groupId == 'com.android.support' && dep.artifactId == 'appcompat-v7' } }
```

Comment:
I think this is broken.  The way I read this: we're assigning the pom.whenConfigured variable to a closure.  Then we re-assign to a different closure right after, dereferencing the first closure.

I think the correct solution involves a single closure with `dep.groupId == 'com.android.support' && (dep.artifactId == 'support-v4' || dep.artifactId == 'appcompat-v7')`

One way to test whether this worked as intended is to run `./gradlew installArchives` and then looking at the local maven repo (on my Mac, under ~/m2/).  The pom file nested in the maven repo lists the required dependencies, and we'd be looking for the absence of support-v4 and appcompat-v7.